EarTrumpet/Properties/Resources.ru.resx

```diff
@@ -118,33 +118,33 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="ContextMenuAboutTitle" xml:space="preserve">
-    <value>About</value>
+    <value>О программе</value>

```

Comment:
Looks like the original `Resources.resx` file was renamed and modified to form this one. The old one needs to stay intact and a **duplicate** made of it to form `Resources.ru.resx` otherwise it'll break the application in the default case. Did you build and run the app after making these changes to make sure they worked?